images/router/egress/squid.conf

```diff
@@ -0,0 +1,9 @@
+# egress-router squid.conf
+# This is a mostly-transparent HTTP/HTTPS proxy; it only exists to
+# redirect requests, not to authenticate them in any way.
+
+http_port 8080
```

Comment:
8080 is fine, but 3128 wouldn't be bad too.  FoxyProxy and the Android proxy config both default to 3128 for the port.